python/docs/conf.py

```diff
@@ -335,7 +335,7 @@
 # If false, no index is generated.
 #epub_use_index = True
 def setup(app):
-    app.add_javascript('copybutton.js')
+    app.add_js_file('copybutton.js')
```

Comment:
`add_javascript` was renamed to `add_js_file` in later sphinx version.

https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.application.Sphinx.add_js_file

Let me know if you prefer to have separate PR for it.

Reply:
I think that's fine